spec.md

```diff
@@ -40,7 +40,7 @@ A platform marketplace MAY expose services from one or many service brokers, and
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to
```

Comment:
extra space before "MAY".  should just search for double-spaces in the entire doc.  :-)